lambda_functions/v1/tests/code_generator/conftest.py

```diff
@@ -0,0 +1,161 @@
+import json
+
+import boto3
+import pytest
+from moto import mock_dynamodb2
+import os
+
+from lambda_functions.v1.functions.lpa_codes.app.api import code_generator
+
+# from lambda_functions.v1.tests.code_generator.test_data import test_data
```

Comment:
Want to get rid of your commented code or is there a reason it's in here?

Reply:
Forgot about it, will fix in next PR